es-design-system/pages/organisms/es-review.vue

```diff
@@ -55,6 +58,7 @@ export default {
                 Mauris vitae tortor mauris. Cras suscipit nibh nec nisi cursus ornare. Maecenas quis turpis sit amet
                 sapien dapibus sollicitudin viverra eu justo. Vivamus posuere metus sit amet purus tempus volutpat.
                 Donec eleifend elit quam.`,
+                response: 'Lorem ipsum',
```

Comment:
Nit: Could we update this to whatever we think a "normal" response length will be?